### DISCUSSION TITLE
Binary nightly snapshot

### DISCUSSION THREAD

**Discussion by johnwason** [tesseract-robotics/tesseract#563] [General]
Can we create a binary nightly snapshot of Tesseract either in tar files or in something like a docker? I have quite a few users now that are having a very hard time compiling from source every time they want to use Tesseract.

**Comment by johnwason** [comment]
We can probably just add an "archive" step to the github actions, so we can download archives of the compiled install space. I am not sure the best way to do this using `industrial-ci`.

**Comment by Levi-Armstrong** [comment]
I just tested out leveraging cpack within cmake to create both Debian on Linux and NuGet package on windows. I created a [PR](https://github.com/Jmeyer1292/opw_kinematics/pull/52) against opw_kinematics and worked very well. Do you think this will work?

**Reply by johnwason** [reply]
How are newer versions of system dependencies handled? Are you installing everything to somewhere in /opt?
